app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
+    <string name="data_sources_description">Please pick a source for charging stations. (Can be changed in the app settings.)</string>
     <string name="data_source_goingelectric">GoingElectric.de</string>
     <string name="data_source_openchargemap">Open Charge Map</string>
-    <string name="data_source_goingelectric_desc">Very good coverage in Germany, Austria and Switzerland and many neighboring countries. Descriptions in German. Community-maintained.</string>
-    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide coverage with varying quality. Descriptions in English or local language. Community-maintained & government open data in some countries (e.g. North America, UK, France, Norway).]]></string>
+    <string name="data_source_goingelectric_desc">Great in germannic countries. Descriptions in German. Community-maintained.</string>
+    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide, with varying quality. Descriptions in English or the local language. Community-maintained and open government-data in some countries (e.g. North America, UK, France, Norway).]]></string>
     <string name="next">next</string>
-    <string name="get_started">Get started</string>
-    <string name="got_it">Got it</string>
+    <string name="get_started">OK</string>
+    <string name="got_it">OK</string>
     <string name="lets_go">Let\'s go</string>
-    <string name="crash_report_text">Sorry, it seems that EVMap has crashed. Please send a crash report to the developer.</string>
-    <string name="crash_report_comment_prompt">You can add a comment below:</string>
+    <string name="crash_report_text">EVMap crashed. Please send a crash report to the developer.</string>
+    <string name="crash_report_comment_prompt">Comment below:</string>
     <string name="powered_by_mapbox">powered by Mapbox</string>
-    <string name="pref_search_provider">Place search provider</string>
-    <string name="pref_search_provider_info"><![CDATA[Data for place search, especially from Google Maps, is relatively expensive. If you use this feature often, please consider making a donation through \"About EVMap -> Donate\".]]></string>
+    <string name="pref_search_provider">Geo-searches</string>
+    <string name="pref_search_provider_info"><![CDATA[Data for geo-searches. Expensive to fetch, especially from Google Maps. Please consider donating through “About” → “Donate”.]]></string>
     <string name="github_sponsors">GitHub Sponsors</string>
-    <string name="donate_desc">Support EVMap\'s development with a one-time donation</string>
-    <string name="github_sponsors_desc">Support EVMap on GitHub Sponsors</string>
+    <string name="donate_desc">Make a one-time donation to support EVMap development</string>
+    <string name="github_sponsors_desc">Support EVMap</string>
```

Comment:
Maybe we should make it something like:

> Support EVMap with one-time or recurring donations

To make it more clear what GitHub Sponsors allows to do?